### DISCUSSION TITLE
About schedule releasing new version

### DISCUSSION THREAD

**Discussion by tranngocbaoduy** [validator/validator#1614] [General]
I'm really sorry but I have a question regarding version release.
We are using your respository to evaluate the quality of our website.
It seems that the official release version is still the version of 2020/06/30 (3 years ago).

Currently I see the version on https://validator.w3.org/nu/ is currently Version: 23.7.8, it seems the update has changed a lot.
Also I recently saw an updated pre-release version but do you have any plans to release the final version?
If so please let us know, thank you.

**Comment by prasad-cm** [comment]
It would be really helpful if we can know the ETA for the next release!!!

**Comment by j9t** [comment]
@tranngocbaoduy, I believe this was solved in the meantime? I couldn’t find the version number on validator.w3.org/nu, but if that and [the npm version](https://www.npmjs.com/package/vnu-jar) are in sync, there have been a few updates, including one just a few hours ago.

Feeling free to close, but follow up if you think this would miss something.
